src/infection-model/seir.tsx

```diff
@@ -307,31 +308,34 @@ export function getAllBracketCurves(inputs: CurveProjectionInputs) {
   ).forEach(([pop, cases, recovered, deaths], index) => {
     // distribute cases across compartments proportionally
 
-    const infectious = cases * pInitiallyInfectious;
+    const activeCases = cases - recovered - deaths;
```

Comment:
Do I need any error checking here to make sure these are all valid numbers and/or that active cases is non-negative?

Reply:
yeah, for safety I would probably clamp it at zero with a call to `Math.max` or something like that ... validation in the UI is still pretty haphazard so we can't necessarily trust that inputs will be sane

Reply:
Cool, clamped the active cases at a minimum of 0 to be safe.